common/app/renderers/DotcomRenderingService.scala

```diff
@@ -176,6 +178,17 @@ class DotcomRenderingService extends GuLogging with ResultWithPreconnectPreload
     post(ws, json, Configuration.rendering.baseURL + "/Front", CacheTime.Facia)
   }
 
+  def getCards(
+      ws: WSClient,
+      cards: List[PressedContent],
+      startIndex: Int,
+  )(implicit request: RequestHeader): Future[Result] = {
+    val dataModel = DotcomCardsRenderingDataModel(cards, startIndex)
+
+    val json = DotcomCardsRenderingDataModel.toJson(dataModel)
```

Comment:
Shall we add the `html` key like it is currently? 
<img width="1777" alt="Screenshot 2022-06-23 at 12 38 12" src="https://user-images.githubusercontent.com/19683595/175291220-dd950c80-7b85-4dae-90c9-a33feadcff6e.png">

Reply:
Currently we're adding this on the DCR side. I feel as though that's the simplest option but definitely open to doing it here instead

Reply:
Perfect, thanks!